clarity.d.ts

```diff
@@ -303,6 +304,14 @@ interface IClarityAssertFailedEventState extends IInstrumentationEventState {
 interface IClarityDuplicatedEventState extends IInstrumentationEventState {
   currentImpressionId: string;
 }
+
+interface IShadowDomInconsistentEventState extends IInstrumentationEventState {
```

Comment:
Added comments

---
In reply to: [131236965](https://github.com/Microsoft/clarity-js/pull/25#discussion_r131236965) [](ancestors = 131236965)